### PR TITLE
bump packages

### PR DESCRIPTION
**Pull request by camrun91**
These should be the same changes that we just pushed to Chase Oaks and a couple of others.

### CODE REVIEW

File: GraceChurch/package.json [hunk 1]
```diff
@@ -41,24 +41,24 @@
...
+    "@apollosproject/ui-media-player": "v2",
+    "@apollosproject/ui-notifications": "v2",
+    "@apollosproject/ui-onboarding": "v2",
+    "@apollosproject/ui-passes": "v2",
+    "@apollosproject/ui-prayer": "v2",
+    "@apollosproject/ui-scripture": "v2",
+    "@apollosproject/ui-test-utils": "v2",
     "@bugsnag/react-native": "^7.14.1",
-    "@gorhom/bottom-sheet": "3.4.1",
+    "@gorhom/bottom-sheet": "^4.1.5",
```
Comment (redreceipt) [inline GraceChurch/package.json:61]: you sure this doesn't break with the provider in `ui-connected` coming from v3?

**Comment by redreceipt** [comment]
did you boot this and make sure it doesn't crash?

File: GraceChurch/package.json (hunk 1 above)
Comment (camrun91) [reply GraceChurch/package.json:61]: Not that I can see. It runs on the sim fine. It looks like in the yarn.lock that 3.4.1 is a dependency of UI-Connected.

Still trying to fully get a grasp of how the dependencies are all interconnected. It looks like on install UI-Connected is pulling in and using v3 

Screen Shot 2022-04-01 at 10 48 26 AM

**Comment by camrun91** [comment]
> did you boot this and make sure it doesn't crash?

Yep on the sim it seems to work on IOS and Android.

**Review by redreceipt — APPROVED** [review]

**Pull request merged**